integration-tests/README.md

```diff
@@ -111,18 +110,12 @@ integration tests:
     go run integration-tests/main.go --snappy-from-branch \
     --revision=-1 --update
 
-To update from *15.04 alpha* to *rolling edge* and then run the integration tests:
-
-    go run integration-tests/main.go --snappy-from-branch \
-    --release=15.04 --channel=alpha \
-    --update --target-release=rolling --target-channel=edge
-
 ### Testing a rollback
 
 With the `--rollback` flag you can flash an old image, update to the latest,
 rollback again to the old image and then run the whole suite on the rolled
-back system. You should use the `release`, `channel`, `revision`, `target-release`
-and `target-channel` flags as when testing an update.
+back system. You should use the `release`, `channel` and `revision` flags as
```

Comment:
Documentation tends to be slightly more pleasant to read (to me?) when it's more informational rather than prescriptive. For example, rather than saying "you should use the flags", I'd generally say "the flags may be used".

The sentence was just changed though, rather than created, so feel free to ignore it.